src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = days * 40;
+  const discountForSeven = 50;
+  const discountForThree = 20;
+
+  if (days >= 7) { // discount is $50 for 7 or more days
+    return cost - discountForSeven;
+  } else if (days >= 3) { // discount is $20 for 3 or more days
+    return cost - discountForThree;
+  }
```

Comment:
no need for `else if` when if statement returns something:

```suggestion
  if (days >= 7) { // discount is $50 for 7 or more days
    return cost - discountForSeven;
  }
  
  if (days >= 3) { // discount is $20 for 3 or more days
    return cost - discountForThree;
  }
```